docs/modules/ROOT/pages/how-tos/mariadbgalera/debug.adoc

```diff
@@ -31,6 +31,26 @@ kubectl -n $INSTANCE_ID logs -c mariadb-galera \
     --prefix -f
 ----
 
+== Crossplane can't reconsilidate users or grants
+
+It might happen that crossplane can't reconsile users or grants on some clusters.
+The logs of the mariadb pod will log something like this:
+
+[source,shell]
+----
+2022-12-15 15:52:14 473 [Warning] WSREP: Ignoring error 'Can't find any matching row in the user table' on query. Default database: ''. Query: 'GRANT ALL ON `49d8e185-8cc0-413e-9d28-2a3bab081300`.* TO 'c89343c6-6367-4a89-9de9-1bfd72643d91'@'%'', Error_code: 1133
+----
+
+This is most likely caused by a bug in MariaDB and the workaround is to increase the `table_definition_cache` of the affected instance.
+
+[source,shell]
+----
+kubectl exec mariadb-0 -c mariadb-galera -it -- bash
```

Comment:
Just put it in one liner, I think it's better

Reply:
What do you mean?

Reply:
you sent me the version to run only in one command, Here you need to do 3 commands or I'm wrong?